micronaut-camunda-bpm-feature/src/main/java/info/novatec/micronaut/camunda/bpm/feature/eventing/PublishHistoryEventHandler.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2021 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package info.novatec.micronaut.camunda.bpm.feature.eventing;
+
+import io.micronaut.context.event.ApplicationEventPublisher;
+import org.camunda.bpm.engine.impl.history.event.HistoryEvent;
+import org.camunda.bpm.engine.impl.history.handler.HistoryEventHandler;
+
+import java.util.List;
+
+/**
+ * Event handler publishing history events as Micronaut Events.
+ */
+public class PublishHistoryEventHandler implements HistoryEventHandler {
```

Comment:
Why not declare this as a singleton? Then you don't need to instantiate it yourself.